### PR TITLE
docs(readme): leave DBeaver to last, not needed to start

### PR DESCRIPTION
**Pull request by LoneRifle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,17 +60,12 @@ Optionally [VSCode](https://code.visualstudio.com/) with extension `ESLint`
...
-  Create database by executing the following in `DBeaver`:
-  ```sql
-  CREATE DATABASE askgov;
-  ```
-
 * Execute `npm run seq-cli db:migrate` to create the tables
   
 * Execute `npm run seq-cli db:seed:all` to seed the database with a sample dataset
 
+* Download a database GUI like [DBeaver](https://dbeaver.io/download/) and connect to the local MySQL server
```
Comment (nauynix) [inline README.md]:
```suggestion
* Download a database GUI like [DBeaver](https://dbeaver.io/download/) and connect to the local MySQL server at `127.0.0.1:3306`, using the username and password in `.env`
```

File: README.md [hunk 2]
```diff
@@ -60,17 +60,12 @@ Optionally [VSCode](https://code.visualstudio.com/) with extension `ESLint`
 
 * Spin up MySQL by running `docker-compose up`
 
-* Download a database GUI like [DBeaver](https://dbeaver.io/download/) and connect to the local MySQL server
-
-  Create database by executing the following in `DBeaver`:
-  ```sql
-  CREATE DATABASE askgov;
```
Comment (nauynix) [inline README.md:67]: We don't need Dbeaver to create the database anymore?

Comment (LoneRifle) [reply README.md:67]: gets initialised by the Docker image:
https://github.com/opengovsg/askgovsg/blob/b7a182b867b3e798641ff430c116c2187c7c65a3/docker-compose.yml#L30

**Review by nauynix — APPROVED** [review]
lgtm

**Pull request merged**